dictionaries/en.dictionary.itop.ui.php

```diff
@@ -656,9 +656,14 @@
 	'UI:UniversalSearch:LabelSelectTheClass' => 'Select the class to search: ',
 
 	'UI:CSVReport-Value-Modified' => 'Modified',
-	'UI:CSVReport-Value-SetIssue' => 'Could not be changed - reason: %1$s',
-	'UI:CSVReport-Value-ChangeIssue' => 'Could not be changed to %1$s - reason: %2$s',
-	'UI:CSVReport-Value-NoMatch' => 'No match',
+	'UI:CSVReport-Value-SetIssue' => 'invalid value for attribute',
```

Comment:
```suggestion
	'UI:CSVReport-Value-SetIssue' => 'Invalid value for attribute',
```

_Also needs to be corrected in other languages_